cmd/porter/mixins.go

```diff
@@ -52,10 +52,10 @@ func BuildMixinInstallCommand(p *porter.Porter) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "install NAME",
 		Short: "Install a mixin",
-		Example: `  porter mixin install helm --url https://cdn.deislabs.io/porter/mixins/helm
-  porter mixin install helm --feed-url https://cdn.deislabs.io/porter/atom.xml
-  porter mixin install azure --version v0.4.0-ralpha.1+dubonnet --url https://cdn.deislabs.io/porter/mixins/azure
-  porter mixin install kubernetes --version canary --url https://cdn.deislabs.io/porter/mixins/kubernetes`,
+		Example: `  porter mixin install helm --url https://cdn.porter.sh/mixins/helm
+  porter mixin install helm --feed-url https://cdn.porter.sh/atom.xml
```

Comment:
So now that we have the CDN rules all fixed, the /atom.xml URL is going to become legacy just to support clients until we can get a PR in place to update Porter to start using /mixins/atom.xml. I think we should stop mentioning this old URL anywhere, and instead with this PR start using the new path.

What do you think?

Reply:
Definitely, thanks for catching.  Updated.